include/groonga/column.h

```diff
@@ -25,6 +25,25 @@ extern "C" {
 
 typedef struct _grn_column_cache grn_column_cache;
 
+/**
+ * \brief Name of the pseudo column `_id`.
+ *
+ * The `_id` pseudo column serves as a unique identifier for each record in a
+ * Groonga table. It is automatically defined for all tables upon creation and
+ * is immutable as long as the record exists. The `_id` value ranges from 1 to
+ * 1,073,741,824. And it also typically increments by one for each new record
```

Comment:
Can we use `GRN_ID_MAX` here?

```suggestion
 * \ref GRN_ID_MAX. And it also typically increments by one for each new record
```

Reply:
fix: [ea09e90](https://github.com/groonga/groonga/pull/2042/commits/ea09e904233bcd4414490aaaec865079fd17a99d) Yes, we can!